src/pages/ImportData.js

```diff
@@ -124,8 +124,26 @@ const ImportDataOverview = ({ header, snapshots, isDataset, snapshotResponses, u
         ])
       : url && div({ style: { fontSize: 16 } }, ['From: ', new URL(url).hostname]),
     div({ style: { marginTop: '1rem' } }, [
-      `The ${isDataset ? 'dataset' : 'snapshot'}(s) you just chose to import to Terra will be made available to you `,
-      'within a workspace of your choice where you can then perform analysis.',
+      Utils.cond(
```

Comment:
Nit: Since there's only two cases, you could do this as a ternary. I think that might be a little clearer in terms of clarity with all the cases/what's being done.

Reply:
Okey-doke!